packages/client/test/client.test.ts

```diff
@@ -33,7 +33,7 @@ describe('client', () => {
     expect(res.data?.createGenericProfile.document.id).toBeDefined()
   }, 30000)
 
-  test('create and query post with comments', async () => {
+  test.skip('create and query post with comments', async () => {
```

Comment:
why do we need to skip this test?  Can you add a comment explaining what's wrong with it and a TODO referencing a linear ticket number where it will be restored?

Reply:
No longer skipped